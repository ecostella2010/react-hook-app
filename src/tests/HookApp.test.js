import React from 'react'; 

import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';

describe('Pruebas en HookApp', () => {

    let wrapper = shallow ( < HookApp />);
    
    beforeEach ( () => {
        wrapper = shallow ( < HookApp /> );
    });

    test('debe mostrar  < HookApp /> correstamente', () => {
        expect (wrapper).toMatchSnapshot();
    });


})
