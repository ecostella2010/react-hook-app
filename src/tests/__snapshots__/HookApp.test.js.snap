// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HookApp debe mostrar  < HookApp /> correstamente 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
